syncserver-common/src/metrics.rs

```diff
@@ -1,5 +1,6 @@
 use std::collections::HashMap;
 use std::net::UdpSocket;
+use std::sync::Arc;
```

Comment:
did functionality changes sneak in here accidentally?

Reply:
A recent change in StatsdClient changed the way that it is supposed to be called. Previously, you would clone the client, but now, it's a singleton that needs to be reference counted.